db/migrate/20170222192610_create_firmwares.rb

```diff
@@ -0,0 +1,14 @@
+class CreateFirmwares < ActiveRecord::Migration[5.0]
+  def change
+    create_table :firmwares do |t|
+      t.string   :name
+      t.string   :build
+      t.string   :version
+      t.datetime :release_date
+      t.bigint   :resource_id
+      t.string   :resource_type
+      t.timestamps
+      t.index %w(resource_id resource_type), :name => "index_firmwares_on_resource_id_and_resource_type", :using => :btree
```

Comment:
I think the `:using` piece is not necessary as that's the default.  Curious why you need the name either.  If that's the default, cause you remove that as well?